# ROSPARAM Yaml file
# use rosparam load 
# Note:: Need to change the path

motion_target_yaml_path: "/home/youliang/catkin_ws/src/robot_arm_workcell_manager/robot_arm_workcell_manager/config/motion_target.yaml" 
marker_tf_path: "/home/youliang/catkin_ws/src/robot_arm_workcell_manager/robot_arm_workcell_manager/config/markers_tf.yaml"
arm_mission_path: "pending"

group_name: "manipulator"
camera_frame_id: "camera" 
flip_marker: true 
dispenser_state_pub_rate: 0.5
motion_pause_time: 1
